chill-hadoop/src/main/java/com/twitter/chill/hadoop/KryoSerializer.java

```diff
@@ -47,7 +47,7 @@ public void serialize(Object o) throws IOException {
         try {
           st.writeObject(o);
           // Copy from buffer to output stream.
-          outputStream.writeInt(st.numOfWrittenBytes());
+          outputStream.writeLong(st.numOfWrittenBytes());
```

Comment:
this is going to write 4 extra bytes for each record, right? Even 4 bytes is more than we need, as we don't expect 4 GB records.

Why not cast this to an int here and save the data (consider serializing Unit with this, which will now take twice as much data as it did before).